script/test

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+npm run markdown:lint &&
```

Comment:
To enhance my understanding: why are you using `&&` to chain these commands?

Reply:
It stops the other commands from running if one fails 👍

Reply:
`set -e` is what you're looking for.

Reply:
Ah, I knew there was a more elegant way of doing this! Ta